CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Changelog
 
+## [1.57](https://github.com/brave/brave-ios/releases/tag/v1.57)
+
+### Web3
+
+- Added search bar to “Select a Token to Send” view. ([#7768](https://github.com/brave/brave-ios/issues/7768))
+- Added “Group by” filter to Portfolio tab in Brave Wallet. ([#7765](https://github.com/brave/brave-ios/issues/7765))
+- Added NFT filters and display settings in the NFT tab for Brave Wallet. ([#7699](https://github.com/brave/brave-ios/issues/7699))
+- Added new UI for the Brave Wallet onboarding flow. ([#7619](https://github.com/brave/brave-ios/issues/7619))
+- Added handling of empty state(s) for Brave Wallet Portfolio filters. ([#7618](https://github.com/brave/brave-ios/issues/7618))
```

Comment:
nit: state(s) -> state. 

No multiple states available so no need to pluralize it